File: SpEditText/src/main/java/com/sunhapper/x/spedit/gif/drawable/InvalidateDrawable.java
package com.sunhapper.x.spedit.gif.drawable;

import com.sunhapper.x.spedit.gif.listener.RefreshListener;

/**
 * Created by sunhapper on 2019/1/25 .
 */
public interface InvalidateDrawable {
    void addRefreshListener(RefreshListener callback);

    void removeRefreshListener(RefreshListener callback);
}

File: SpEditText/src/main/java/com/sunhapper/x/spedit/gif/drawable/InvalidateDrawable.kt
package com.sunhapper.x.spedit.gif.drawable

import com.sunhapper.x.spedit.gif.listener.RefreshListener

/**
 * Created by sunhapper on 2019/1/25 .
 */
interface InvalidateDrawable {
    var mRefreshListeners: MutableCollection<RefreshListener>
    fun addRefreshListener(callback: RefreshListener)
    fun removeRefreshListener(callback: RefreshListener)
    fun refresh()
}
